contrib/podmanimage/machine/Containerfile

```diff
@@ -0,0 +1,26 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+RUN dnf -y install podman subscription-manager crun crun-wasm crun-krun chrony; dnf -y clean all
+RUN dnf -y install skopeo buildah; dnf -y clean all
+RUN systemctl enable podman.service
+RUN printf "\nconfdir /etc/chrony.d\n" >> /etc/chrony.conf
+
+COPY 10-inotify-instances.conf /usr/etc/sysctl.d/10-inotify-instances.conf
+COPY 50-podman-makestep.conf /usr/etc/chrony.d/50-podman-makestep.conf
```

Comment:
Also what IMO is *way* nicer to read and maintain is just to have:
```
COPY rootfs/etc etc
COPY rootfs/usr usr
```

And then just drop your files in those directories in git.

There's [one example of this here](https://github.com/coreos/layering-examples/blob/9329023bee5c188466f65100064fb917f36af39c/autoupdate-unit/Containerfile#L4)

Reply:
I just used usr and etc, but this was a good idea.